lib/dry/schema/error_compiler.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Dry
+  module Schema
+    # Compiles rule results AST into some type of errors
+    #
+    # @api private
+    class ErrorCompiler
+      # @api private
+      def call(_ast)
+        raise NotImplementedError
+      end
+
+      # @api private
+      def with(*args)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `with(*)` if you want the method to accept any arguments but don't care about them.](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764391)